omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -794,8 +794,6 @@
 default['private_chef']['keepalived']['vrrp_instance_virtual_router_id'] = "1"
 default['private_chef']['keepalived']['vrrp_instance_priority'] = "100"
 default['private_chef']['keepalived']['vrrp_instance_advert_int'] = "1"
-# TODO mp 2017-03-10 this one too
-default['private_chef']['keepalived']['vrrp_instance_password'] = "sneakybeaky"
```

Comment:
Ok, I was mislead by [this](https://github.com/chef/chef-server/blob/master/omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb#L484-L485) then...